internal/terraform/lang/resource_block.go

```diff
@@ -81,7 +81,9 @@ func (r *resourceBlock) Labels() []*ParsedLabel {
 	if r.labels != nil {
 		return r.labels
 	}
-	r.labels = parseLabels(r.BlockType(), r.labelSchema, r.hclBlock.Labels)
+	labels, _ := ParseLabels(r.tokens, r.labelSchema)
```

Comment:
Same as above, could you add a comment here?

Reply:
Given that `ParseLabels` doesn't actually need to return any error anymore I will just remove the err from the return values.